photon-server/src/main/java/org/photonvision/server/RequestHandler.java

```diff
@@ -893,59 +927,81 @@ public static void onNukeOneCamera(Context ctx) {
 
             ctx.status(200);
         } catch (IOException e) {
-            // todo
-            logger.error("asdf", e);
+            logger.error("Failed to delete camera", e);
             ctx.status(500);
+            ctx.result("Failed to delete camera");
+            return;
         }
     }
 
     public static void onActivateMatchedCameraRequest(Context ctx) {
         logger.info(ctx.queryString());
+        try {
+            CommonCameraUniqueName request =
+                    kObjectMapper.readValue(ctx.body(), CommonCameraUniqueName.class);
+            String cameraUniqueName = request.cameraUniqueName;
 
-        String cameraUniqueName = ctx.queryParam("cameraUniqueName");
-
-        if (VisionSourceManager.getInstance().reactivateDisabledCameraConfig(cameraUniqueName)) {
-            ctx.status(200);
-        } else {
-            ctx.status(403);
+            if (VisionSourceManager.getInstance().reactivateDisabledCameraConfig(cameraUniqueName)) {
+                ctx.status(200);
+            } else {
+                ctx.status(403);
+            }
+        } catch (IOException e) {
+            ctx.status(401);
+            logger.error("Failed to process activate matched camera request", e);
+            ctx.result("Failed to process activate matched camera request");
+            return;
         }
-
-        ctx.result("Successfully assigned camera with unique name: " + cameraUniqueName);
     }
 
+    public record AssignUnmatchedCamera(PVCameraInfo cameraInfo) {}
```

Comment:
Should these records all live in one spot? Or do they make sense closer tied to their handlers?

Also, why public?